modules/component-store/src/component-store.ts

```diff
@@ -32,6 +34,14 @@ export interface SelectConfig {
   debounce?: boolean;
 }
 
+export interface OnStoreInit {
```

Comment:
Should we add JS Doc comments here so it looks nice on the API documentation?

Reply:
Yep. Will add that also